### ISSUE TITLE
[Dev UI] Better handling of long text for input

### ISSUE THREAD

**Issue by DenisVCode** [firebase/genkit#1577]
## Overview
Better way of supporting long input text when input is just pure string (but probably also applies to nested longer text strings inside JSON)

### User goal(s)
Be able to just paste long block of text inside double quotes and it just works. Currently I have to do JSON.stringify and copy paste that output as a helper. 

## Requirements
- Detect that longer text is being put inside the text editor that would otherwise break the JSON.parse and reformatt it first. 

### Designs

![Image](https://github.com/user-attachments/assets/8af6774a-a3f0-4c8c-a938-c58a4f6a1ef8)

## Notes

Related Discord link: https://discord.com/channels/1255578482214305893/1255626681146806434/1323298317865717761

**Comment by MichaelDoyle** [comment]
One final bit of context...

This is a Monaco "editor" that is validating the contents based on a JSON schema. For "objects", generally speaking, people are OK with this. For plain text "string" inputs, it really has a poor user experience, because most people don't expect they need to "format" the string as "json" (in this case, single line, double quoted).

I am really starting to think we should not use an "editor" in the plain string case. But maybe there is a technical reason to keep doing so, that I cannot think of. @tonybaroneee @mjchristy @ssbushi.

We could reformat and remove newlines. There is also the funny business of making sure its double quoted (and then inline quotes are escaped, etc). I think it might get complicated reasonably quickly.

**Comment by tonybaroneee** [comment]
> One final bit of context...
> 
> This is a Monaco "editor" that is validating the contents based on a JSON schema. For "objects", generally speaking, people are OK with this. For plain text "string" inputs, it really has a poor user experience, because most people don't expect they need to "format" the string as "json" (in this case, single line, double quoted).
> 
> I am really starting to think we should not use an "editor" in the plain string case. But maybe there is a technical reason to keep doing so, that I cannot think of. @tonybaroneee @mjchristy @ssbushi.
> 
> We could reformat and remove newlines. There is also the funny business of making sure its double quoted (and then inline quotes are escaped, etc). I think it might get complicated reasonably quickly.

Yeah, fair point. If the input schema is a simple primitive (e.g. string, number), perhaps monaco causes more potential issues than it's worth.
